jcommune-service/src/test/java/org/jtalks/jcommune/service/transactional/TransactionalUserContactsServiceTest.java

```diff
@@ -104,6 +104,26 @@ public void saveEditedContactsShouldUpdateEditedContacts() throws NotFoundExcept
     }
 
     @Test
+    public void saveEditedContactsWithNotEmptyValue() throws NotFoundException {
```

Comment:
Test name needs to contain the expected result. E.g. `saveEditedContacts_mustActuallSaveThem`